packages/server/src/main/java/software/uncharted/terarium/hmiserver/service/tasks/TaskService.java

```diff
@@ -367,7 +367,7 @@ private void onTaskResponseOneInstanceReceives(final Message message) {
 
 				notificationService.createNotificationEvent(resp.getId(), event);
 			} catch (final Exception e) {
-				log.error("Failed to persist notification event for for task {}", resp.getId(), e);
+				log.error("Failed to persist notification event for for task {}", resp.getId());
```

Comment:
Are we logging those exception somewhere else, ot the `resp.getId()` is the only info of interest here?

Reply:
They just add noise, task id is enough.